src/scripts/TranscriptsTrack.js

```diff
@@ -1459,69 +2959,156 @@ const TranscriptsTrack = (HGC, ...args) => {
+                          : "non-essential";
+                        return `
+                          <div style="text-align: center; padding:10px;">
+                            <div>
+                              <img class="fit-picture" 
+                                  src="${aa.image}" 
+                                  style="width:125px;height:125px"
+                                  alt="${aa.name}">
+                            </div>
+                            <div style="text-transform: uppercase; padding-top:6px;">
+                              <b>${aa.name}</b>
+                            </div>
+                            <div style="padding-top:0px;">
+                              <i>${aa.property}, ${essential}</i>
+                            </div>
+                            <div style="font-size: 10px;">
+                              ${aa.codons.join(", ")}
+                            </div>
+                          </div>
+                        `;
+                      }
+                    }
+                  } 
+                  else {
                     return `
-                      <div style="text-align: center; padding:10px;">
-                        <div style="text-transform: uppercase;">
-                          <b>${aa.name}</b>
-                        </div>
-                        <div style="font-size: 10px;">
-                          ${aa.codons.join(", ")}
-                        </div>
+                      <div>
+                        <div><b>Transcript: ${transcript.transcriptName}</b></div>
+                        <div>Position: ${transcript.chromName}:${transcript.txStart}-${transcript.txEnd}</div>
+                        <div>Strand: ${transcript.strand}</div>
+                        <div>${formattedOverlap}</div>
                       </div>
                     `;
                   }
+                }
+                break;
+              case "boxplot":
+                {
+                  const bed12FieldsObj = {
+                    chrom: transcript.chromName,
+                    start: transcript.txStart,
+                    end: transcript.txEnd,
+                    id: transcript.rawTranscriptName,
+                    score: transcript.importance,
+                    strand: transcript.strand,
+                    thickStart: transcript.startCodonPos,
+                    thickEnd: transcript.stopCodonPos,
+                    itemRGB: (transcript.itemRgb !== '.') ? transcript.itemRgb : '0,0,0',
+                    blockCount: transcript.boxBlockCount,
+                    blockSizes: transcript.boxBlockLengths,
+                    blockStarts: transcript.boxBlockStarts,
+                  };
+
+                  const bed12Output = this.formattedBED12HTML(bed12FieldsObj);
+
+                  if (bed12Output.length === 0) {
+                    const hc = document.getElementsByClassName('higlass-main-content')[0];
```

Comment:
I guess that only works, when there is only one Higlass display on the page. It's fine with me. I was just wondering if there is a simple way to get the "current" display.

Reply:
I haven't run into a situation with more than one higlass display for my apps, which is likely why I did it this way. 

Is this something that Pete or Fritz might know more about?